mkdocs/docs/HPC/teaching_training.md

```diff
@@ -0,0 +1,108 @@
+
+You provide us with the vsc account(s) for the moderator(s),
+the moderators can manage the members and moderators themself afterwards.
+
+[^TODO]: do we support initial list of participants?
+
+[^TODO]: we control the groupname prefix? or have them all use the same one (unprotected). incl some convntion like academci year
+
+
+### UGent course id
+
+[^TODO]: check terminology
+
+When teaching a registered UGent course, we can offload the management of the vsc accounts from the students and
+the teachers. We do this by creating a vsc account for each registered student that is linked to the course id
+(if they do not have an account already). And we then populate the group with the vsc accounts of the students
+and the list of moderators. Using this method, you cannot manage the members or moderators yourself:
+group members unknown to the course will be considered unregistered students and will be removed from the group.
+
+The created accounts will be accounts without an ssh-key. This allows the students to use e.g. the portal, but if
+they require ssh access to the infrastructure, they will have to add one themself.
+
+[^TODO]: what does the sync do? can moderators add moderators themself?
+
+## Start and end date
+
+We require an end date to perform any required cleanup of groups and optional data, apps or reservations.
+
+[^TODO]: can we do some automated cleanup for ugent courses? when the course disappears from AD?
+
+[^TODO]: we need a maximum enddate? max one year after the request?
+
+The start date will be `best-effort`, based on the load of the support team and the requirements you have.
+The sooner you make the request, the better.
+
+[^TODO]: do we mention some actual ETA here? like one month upfront?
+
+[^TODO]: when do course of the next academic year appear?
+
+## Share
+
+When required, we can create a directory that is writeable by the moderators and accessible by all
+members and moderators. This directory can be used to share e.g. input files with the students.
+
+The data policy for this share is the following: on the enddate, the directory will be made unaccessible
+nby anyone, and one year after the enddate it will be removed. When the moderators require anything from
+this expired share, they will have to make a request via the support page. We assume that you always have
+your own copy of the course data as a starting point for a next course.
+
+[^TODO]: to avoid extra support tickets, we can also make the dir readonly and only accessible by the moderators.
+
+## Reservation
+
+When the resources offered by the interactive cluster are not sufficient for the course, you will need to request
+a reservation as well. Indicate the number of nodes and which cluster you need and when you need them
+(e.g. the dates of each course). Be aware that students will have no access to the reservation outside the courses.
+This might be relevant when requesting a custom application.
```

Comment:
No idea what this means.

Maybe also mention that we strongly prefer not creating/managing a reservation if it can be avoided.

Maybe also mention that we can temporarily ease user limits in interactive cluster a bit?